packages/mdc-shape/README.md

```diff
@@ -36,74 +35,50 @@ applying angled corners to unelevated surfaces.
 npm install @material/shape
 ```
 
-## Basic Usage
+## Usage
```

Comment:
Discussed offline: let's bring this back in line with the readme standards/template.